images/runtime/node/4.4/Dockerfile.base

```diff
@@ -33,6 +33,10 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && tar -xJf "node-v$NODE_VERSION-linux-x64.tar.xz" -C /usr/local --strip-components=1 \
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
+CMD [ "node" ]
+
+FROM node-original AS main
```

Comment:
motivation for this change? was it to have a clear separation of what the official image content was vs. what we added more?

Reply:
Yes, to separate the official image